src/Feature.cpp

```diff
@@ -119,12 +119,13 @@ const std::vector<Feature*>& Feature::GetFeatureList()
 	};
 
 	static std::vector<Feature*> featuresVR = {
-		DynamicCubemaps::GetSingleton(),
```

Comment:
This will cause unnecessary conflicts. Minimize useless changes if possible.

Reply:
Reverted unnecessary change (Just wanted to more clearly see which feature are left for VR, but can do that later, maybe even create a common list that we add feature enabled for both to make it easier to see)